internal/commands/scarealtime/sca-realtime_test.go

```diff
@@ -57,3 +57,39 @@ func TestRequiredProjectDir(t *testing.T) {
 	err := cmd.Execute()
 	assert.Error(t, err, "Provided path does not exist: "+invalidProjectPath, err.Error())
 }
+
+func TestCreateDependecyMapFromDependecyResolution_Success(t *testing.T) {
```

Comment:
Change test function name according to convention "TestFunc_conditions_expectedResult"